make/README.md

```diff
@@ -5,9 +5,9 @@ This folder contains the GCC build system for the nRF5x platforms. It
 originally comes from
 [hlnd/nrf51-pure-gcc-setup](https://github.com/hlnd/nrf51-pure-gcc-setup).
 
-We develop on Linux. Original files still exist for Windows, but have not been
-tested in some time. Feel free to submit a pull request if you find something
-that doesn't work.
+We develop on Linux. This also works on Windows in either of the following ways:
+1. Build on WSL (Windows Subsystem for Linux), and `flash` with WSL.
+2. Build with SES (Segger Embedded Studio) and `flash` with WSL. Note: For this you will have to manually move the `.hex` file generated from SES to `build/` directory. For e.g. `mv blinky/pca10040/s132/ses/Output/Release/Exe/blinky_pca10040_s132.hex  blinky/_build/blink_sdk16_blank.hex`.
```

Comment:
What is `blinky`? It would be better to give instructions for an application in the `apps/basic/` directory.

Reply:
For the `nrf52x-base/apps/basic/blink` it works as is. You just have to `make flash`.
The second option is if you are using SES from the SDK.

Reply:
regardless "blinky" isn't an application we have in the repo

Reply:
I see. Is there really any reason to list the SES option at all? It doesn't really let you build nrf52x-base applications,  just apps in the SDK, which aren't our purview.

Reply:
I think there is value to provide minimal support to SES.
@nealjack Good point. Thanks.

Reply:
But there isn't really support for SES. To support SES would mean taking an application from the `apps/` folder, compiling it with SES, and then flashing it. But that is not going to work.